### DISCUSSION TITLE
Monochrome Icon Discussion

### DISCUSSION THREAD

**Discussion by borgmanJeremy** [flameshot-org/flameshot#2152] [General]
What do people think about packaging the monochrome icon as the default for version 11? On the MacOs release we 100% will use the monochrome icon.

Curious what the community wants for Linux / Windows.

**Comment by mmahmoudian** [comment]
We kinda briefly touched upon this issue on #2017 and [I suggested to make it customizable](https://github.com/flameshot-org/flameshot/issues/2017#issuecomment-953585626). I personally use monochrome dark icon because of my KDE Plasma theme, but I think we should keep our original logo in the application. The name and the logo are the only identifiers we have for Flameshot. To some extent the logo is part of the project's identity.
